wrims/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+repositories {
+    mavenLocal()
```

Comment:
Usually, this is a bad idea. If you need to link multiple Gradle projects together, there are other methods (composite builds). Having `mavenLocal` in a buildscript can cause really weird issues in many cases.

Reply:
removed